### PR TITLE
Add query graph visuals

### PR DESCRIPTION
**Pull request by frouioui**
https://github.com/user-attachments/assets/db94efc8-58f6-47ff-9d92-8fab80a2a655

### CODE REVIEW

File: go/summarize/force-graph.go [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+func renderQueryGraph(s *Summary) {
+	data := createForceGraphData(s)
+
+	// Start the HTTP server
+	http.HandleFunc("/", func(w http.ResponseWriter, _ *http.Request) {
+		serveIndex(w, data)
+	})
+
+	port := "1010"
+	fmt.Printf("Server started at http://localhost:%s\n", port)
```
Comment (systay) [inline go/summarize/force-graph.go:76]: maybe add something about how to exit the app? ctrl-c, right?

File: go/summarize/force-graph.go [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+func renderQueryGraph(s *Summary) {
+	data := createForceGraphData(s)
+
+	// Start the HTTP server
+	http.HandleFunc("/", func(w http.ResponseWriter, _ *http.Request) {
+		serveIndex(w, data)
+	})
+
+	port := "1010"
```
Comment (systay) [inline go/summarize/force-graph.go:75]: shouldn't we use a random port instead?

File: go/summarize/force-graph.go [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+	fmt.Printf("Server started at http://localhost:%s\n", port)
+	// nolint: gosec,nolintlint // this is all ran locally so no need to care about vulnerabilities around timeouts
+	if err := http.ListenAndServe(":"+port, nil); err != nil {
```
Comment (systay) [inline go/summarize/force-graph.go:78]: I think we should use `127.0.0.1:<port>` here and not just `:<port>`. No need to open it to anyone outside of localhost, right?

**Review by systay — APPROVED** [review]

File: go/summarize/force-graph.go (hunk 3 above)
Comment (frouioui) [reply go/summarize/force-graph.go:78]: Fixed via https://github.com/vitessio/vt/pull/74

File: go/summarize/force-graph.go (hunk 2 above)
Comment (frouioui) [reply go/summarize/force-graph.go:75]: Fixed via https://github.com/vitessio/vt/pull/74

File: go/summarize/force-graph.go (hunk 1 above)
Comment (frouioui) [reply go/summarize/force-graph.go:76]: Fixed via https://github.com/vitessio/vt/pull/74

**Pull request merged**